tests/tom-swifty-test/Compiler.cs

```diff
@@ -41,6 +41,10 @@ public class Compiler {
 
 		static string kSwiftCustomBin = PosixHelpers.RealPath (SOM_PATH is null ? Path.Combine (kSwiftCustomDirectory, "bin/") : FindPathFromEnvVariable ("bin/swift/bin/")) + "/";
 		static string kSwiftCustomLib = PosixHelpers.RealPath (SOM_PATH is null ? Path.Combine (kSwiftCustomDirectory, "lib/swift/macosx/") : FindPathFromEnvVariable ("bin/swift/lib/swift/macosx/"));
+
+		static string kSystemBin = "/usr/bin/";
+		static string kSystemLib = "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift-5.0/macosx";
```

Comment:
Xcode.app will give issue, why not using the Xcode12_0_0.app path? And again, maybe again get it from the makefile.

Reply:
PRs welcome: https://github.com/xamarin/binding-tools-for-swift/issues/458